docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,154 +3,306 @@ title: JavaScript Quickstart Guide
-[devtools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) extension.
+## Add basic functionalities
 
-`hmsActions` will help us perform actions such as joining the room, mute our audio and send messages.
+In this section, we will walk you through instructions to add basic functionalities for a video conferencing application so that you can test the app locally to see how two or more peers can join this app to communicate with each other.
 
 ### Concepts
 
--   `Room`: When we join a conference call, the participants are said to be in a video call `room`.
+Let's look at some basic concepts to understand how a video conferencing app works.
+
+-   `Room`: A `room` is a virtual space where one or more peers communicate with each other. It is a central entity that provides real-time communication capabilities for any video conferencing or live streaming session. To allow users to join a 100ms video conferencing or a live streaming session inside your app, you first need to create a room.
 -   `Peer`: A participant in the video call. You are the `local peer` while others are `remote peers`.
 -   `Track`: Media. There are two types of track a peer can have - audio and video.
 
-## Joining a room
+### Initialize the SDK
+
+Let us start with initializing the libraries. We need an instance of [HMSStore](/api-reference/javascript/v2/interfaces/HMSStore) and [HMSActions](/api-reference/javascript/v2/interfaces/HMSActions) to get started. Copy the code snippet below and paste it into your project.
+
+```jsx:index.js section=InitializingTheSDK sectionIndex=1
+import {
+  HMSReactiveStore,
+  selectIsLocalAudioEnabled,
+  selectIsLocalVideoEnabled,
+  selectPeers,
+  selectIsConnectedToRoom
+} from "@100mslive/hms-video-store";
+
+// Initialize HMS Store
+const hmsManager = new HMSReactiveStore();
+hmsManager.triggerOnSubscribe();
+const hmsStore = hmsManager.getStore();
+const hmsActions = hmsManager.getHMSActions();
+```
+
+-   [HMSStore](/api-reference/javascript/v2/interfaces/HMSStore) will hold the complete state of the application such as details of all the participants. We can also visualize this state at any time using the [DevTools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) extension.
+-   [HMSActions](/api-reference/javascript/v2/interfaces/HMSActions) will help us perform actions such as joining the room, mute our audio and send messages.
+
+### Initialize the HTML elements
 
-To join a room (a video call), we need to call the `join` method on `hmsActions` and it requires us to pass a `config` object.
+Now, let's initialize all the HTML elements required such as tile for local and remote peers, join and leave button, mute and unmute button, etc.
+
+```jsx:index.js section=InitializingTheSDK sectionIndex=2
+// HTML elements
```

Comment:
we should have html section also here showing all the elements we are accessing

Reply:
added